redash/destinations/email.py

```diff
@@ -46,7 +46,7 @@ def notify(self, alert, query, user, new_state, app, host, options):
         logging.debug("Notifying: %s", recipients)
 
         try:
-            alert_name = alert.name.encode("utf-8", "ignore")
+            alert_name = alert.name
```

Comment:
just drop this variable and use alert.name in line 57 :)

Reply:
https://github.com/getredash/redash/pull/4627/commits/0a8663402921647531bf813d46b308d0f73490a6